bulk_query_gen/metaquery.go

```diff
@@ -0,0 +1,10 @@
+package bulk_query_gen
+
+type Metaquery interface {
+	StandardMetaquery(Query)
+	Dispatch(int) Query
+}
+
+func MetaqueryDispatchAll(d Metaquery, iteration int, q Query, scaleVar int) {
```

Comment:
Why bother with the iteration and scaleVar arguments when they're not used?

Reply:
I was basically using other similar functions as a template, but yeah there's not any reason to have those extra args, so I'll remove them!